Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>DJANGO_DEBUG=True\r\nDJANGO_SECRET_KEY='django-insecure-pjp%k$_v6)y@r+-*0mblxn@*-lb+^lgbyr$1p-1goks#ino$ev'\r\n# DATABASE_URL='postgresql://neondb_owner:qWDwy7UgH6Vo@ep-snowy-bread-a55incku.us-east-2.aws.neon.tech/neondb?sslmode=require'\r\nDATABASE_URL='postgresql://neondb_owner:qWDwy7UgH6Vo@ep-black-pine-a5lzj7jo.us-east-2.aws.neon.tech/neondb?sslmode=require'\r\nEMAIL_HOST='smtp.gmail.com'\r\nEMAIL_PORT='587'\r\nEMAIL_USE_TLS=True\r\nEMAIL_USE_SSL=False\r\nEMAIL_HOST_USER=\"eng.thiagofagundes@gmail.com\"\r\nEMAIL_HOST_USER=\"avpr ehxr fwzt crub\"\r\nADMIN_USER_NAME=\"thiago.fagundes\"\r\nADMIN_USER_EMAIL=\"operandoumsaas@gmail.com\"\r\nSTRIPE_SECRET_KEY=\"sk_test_51PsSpPDwFhGKuGWxr9MGiseAf1nm3cLkxqCcJNVhIOsnQx5q8X4WyhFDvWDfcTil3ept2dewbam8Sz6T6nh1bF6E00p1q6QgP9\"\r\nBASE_URL='http://127.0.0.1:8000/'
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
--- a/.env	(revision a2ffdf4e4fd50ae75e9007636cb8d946d0211f1a)
+++ b/.env	(date 1724942581535)
@@ -2,12 +2,14 @@
 DJANGO_SECRET_KEY='django-insecure-pjp%k$_v6)y@r+-*0mblxn@*-lb+^lgbyr$1p-1goks#ino$ev'
 # DATABASE_URL='postgresql://neondb_owner:qWDwy7UgH6Vo@ep-snowy-bread-a55incku.us-east-2.aws.neon.tech/neondb?sslmode=require'
 DATABASE_URL='postgresql://neondb_owner:qWDwy7UgH6Vo@ep-black-pine-a5lzj7jo.us-east-2.aws.neon.tech/neondb?sslmode=require'
-EMAIL_HOST='smtp.gmail.com'
+ACCOUNT_EMAIL_VERIFICATION="none"
+EMAIL_HOST='smtp.sendgrid.net'
 EMAIL_PORT='587'
 EMAIL_USE_TLS=True
 EMAIL_USE_SSL=False
-EMAIL_HOST_USER="eng.thiagofagundes@gmail.com"
-EMAIL_HOST_USER="avpr ehxr fwzt crub"
+EMAIL_HOST_USER="apikey"
+EMAIL_HOST_PASSWORD='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'
+DEFAULT_FROM_EMAIL = "eng.thiagofagundes@gmail.com"
 ADMIN_USER_NAME="thiago.fagundes"
 ADMIN_USER_EMAIL="operandoumsaas@gmail.com"
 STRIPE_SECRET_KEY="sk_test_51PsSpPDwFhGKuGWxr9MGiseAf1nm3cLkxqCcJNVhIOsnQx5q8X4WyhFDvWDfcTil3ept2dewbam8Sz6T6nh1bF6E00p1q6QgP9"
